src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -29,6 +30,8 @@ public class DocumentDatabase : IResourceStore
         private readonly CancellationTokenSource _databaseShutdown = new CancellationTokenSource();
         public readonly PatchDocument Patch;
 
+		private readonly HttpJsonRequestFactory _httpJsonRequestFactory = new HttpJsonRequestFactory(16);
```

Comment:
Why is this here? I think this is only used in the replication, no? So why create it on the doc db?